### ISSUE TITLE
Normalize naming conventions

### ISSUE THREAD

**Issue by X3msnake** [lexelby/inkstitch#126]
We need to revise the naming used in the menus. 

As it is, it is prone to confusion.

For example a Stroke should be named Running Stitch
 At the moment we have

## STITCHES

STITCH NAME | DESCRIPTION | MENU NAME | CONDITION
--|--|--|--
Running Stitch | Single line with consecutive stitches | Stroke | Path with dotted stroke or width <= 0.5
Simple Satin | constant width ZigZag perpendicular to path | Stroke | Path with solid line > 0.5
Satin Column (equal nodes) | Variable width ZigZag guided by node pairs | Satin Column | 2 combined paths each with the same number of nodes 
Satin Column (ladder) | Variable width ZigZag guided by intersecting lines| Satin Column | 2 combined paths, each with any number of nodes where ZigZag direction is given by intersecting lines
Fill | Parallel stitches to cover large areas | Auto Fill / Fill (Manual?) | Closed path with fill colour

## UNDERLAYS

UNDERLAY NAME | DESCRIPTION | MENU NAME | CONDITION
--|--|--|--
Center Run | straight stitches running down the center of the satin column | Center Walk Underlay | Only on Satin
Contour | straight stitches running inside the edge of each side of the satin column giving a "clean" edge | Countour Underlay | Only on Satin 
Zig-Zag | widely spaced zig-zag of long stitches running under the satin column at low density at different angle than the path of the stitches above | Zig-Zag Underlay | Only on Satin
Fill | Less dense block of fill stitches that runs under the top stitching at a different angle, often set perpendicular | Auto Fill Underlay | Only on Fill

**Comment by wwderw** [comment]
This is the biggest bug bear of commercial software.  Some are universal, but not all.  I've seen fill stitch referred to as fill, step, tatami etc.

**Comment by lexelby** [comment]
Yes, normalizing the naming is a good idea.  I named stuff as I learned machine embroidery and I didn't always get things right.

@AmberSuzanne this may be relevant to your interests :)

For the terms "center walk underlay" and "contour underlay", I got those from Mr Xstitch.

**Comment by lexelby** [comment]
Yup.  I've seen fill referred to as "trapunto fill" if the travel between sections is always done on the outline.  That's the only kind Ink/Stitch does.

**Comment by X3msnake** [comment]
![image](https://user-images.githubusercontent.com/11083514/37880387-8b1d77aa-307f-11e8-93e8-f9f792e0a70e.png)

@lexelby 
I would change Stitch Plan Layer Name to something like Stitchout Preview
